### PR TITLE
{Packaging}: Bump msrest to 0.6.21

### PR DESCRIPTION
**Pull request by houk-ms**
**Description**

This PR bumps `msrest` to `0.6.21`, as `azure-keyvault-administration==4.0.0b3` depends on `msrest>=0.6.21`. The keyvault package is used for the support of keyvault custom role definition feature.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@jiasli for awareness

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/requirements.py3.windows.txt [hunk 1]
```diff
@@ -105,7 +105,7 @@ jsmin==2.2.2
 knack==0.8.0rc2
 MarkupSafe==1.1.1
 mock==4.0.2
-msrest==0.6.18
+msrest==0.6.21
```
Comment (yungezz) [inline src/azure-cli/requirements.py3.windows.txt:108]: no change need in setup.py?

**Review by yungezz — APPROVED** [review]

File: src/azure-cli/requirements.py3.windows.txt (hunk 1 above)
Comment (houk-ms) [reply src/azure-cli/requirements.py3.windows.txt:108]: yes, msrest version is not specified explicitly in setup.py, so `azure-keyvault-administration==4.0.0b3` there will make sure `msrest>=0.6.21`

**Review by fengzhou-msft — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
Please pay attention to the results of scheduled live test on Friday morning.

**Pull request merged**